src/common/front-end-routing/routes/preferences/navigate-to-preferences.injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import navigateToAppPreferencesInjectable from "./app/navigate-to-app-preferences.injectable";
+
+const navigateToPreferencesInjectable = getInjectable({
```

Comment:
why do we need this? Can't `navigateToAppPreferencesInjectable` be used directly?

Reply:
If you want to navigate to first page of preferences you should use `navigateToPreferences`. If you want to navigate to specific tab in preferences, you should use e.g. `navigateToAppPreferences`. 

Motivation is to allow change of mind in future without need for multiple modifications.

Reply:
You mean so one day `navigateToAppPreferences` may take optional param(s) to select the tab?

Reply:
No. `navigateToAppPreferences` doesn't need any parameters. However we might change our mind that `navigateToPreferences` doesn't go to app preferences. One day it might go to editor preferences and for that change we would need to change lot of file instead of one.

Implementation for `navigateToPreferences` could even be "go to first tab in preferences" and it shouldn't need to know which tab is first. In that case `navigateToPreferences` doesn't have to change at all if someday we change order of tabs in preferences.